chain/rust/src/assets.rs

```diff
@@ -26,15 +26,45 @@ pub enum AssetArithmeticError {
 }
 
 /// Bundle of assets within range of T, grouped by PolicyID then AssetName
-#[derive(Clone, Debug, Default, PartialEq, Hash, serde::Deserialize, serde::Serialize, schemars::JsonSchema)]
+#[derive(Clone, Default, PartialEq, Hash, serde::Deserialize, serde::Serialize, schemars::JsonSchema)]
 pub struct AssetBundle<T>(OrderedHashMap<PolicyId, OrderedHashMap<AssetName, T>>);
 
+impl<T: std::fmt::Debug> std::fmt::Debug for AssetBundle<T> {
```

Comment:
I'm not sure if we want this to be the default `Default` implementation or instead a separate method since it can truncate data (the middle parts of policy ids and possibly asset names). It was definitely far more readable when debugging these tests vs the default implementation though.